### PR TITLE
latte-dock: Update to 0.10.8

### PR DESCRIPTION
**Pull request by LupertEverett**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/latte-dock/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'latte-dock'
 pkgname=latte-dock
-version=0.10.4
+version=0.10.7
```
Comment (ericonr) [inline srcpkgs/latte-dock/template:3]: .8 is out

Comment (LupertEverett) [reply srcpkgs/latte-dock/template:3]: Oh! Thanks for the heads up! I'll take a look at that asap

No idea how I can update this pull request though, I am relatively new to git.

Comment (LupertEverett) [reply srcpkgs/latte-dock/template]: Updated, and changed the relevant parts of PR accordingly.

File: srcpkgs/latte-dock/template [hunk 2]
```diff
@@ -14,4 +14,4 @@ license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://invent.kde.org/plasma/latte-dock/"
 changelog="https://invent.kde.org/plasma/latte-dock/-/blob/master/CHANGELOG.md"
```
Comment (paper42) [inline srcpkgs/latte-dock/template:15]:
```suggestion
changelog="https://invent.kde.org/plasma/latte-dock/-/raw/master/CHANGELOG.md"
```

**Review by paper42 — COMMENTED** [review]
rebase on top of master and make sure this PR contains only one commit

**Comment by LupertEverett** [comment]
> rebase on top of master and make sure this PR contains only one commit

I think I messed up there... I should better open a new PR. Sorry, still quite inexperienced with git.

**Comment by ericonr** [comment]
You never want to use merge for void-packages. Use `git rebase` and `git pull --rebase` instead.

**Pull request closed without merging**